learningresources/search_indexes.py

```diff
@@ -0,0 +1,25 @@
+"""
+Index information for Haystack.
+
+http://django-haystack.readthedocs.org/en/latest/tutorial.html
+"""
+
+from haystack import indexes
+
+from learningresources.models import LearningResource
+
+
+class LearningResourceIndex(indexes.SearchIndex, indexes.Indexable):
+    """
+    Index configuration for the LearningResource model.
+    """
+    text = indexes.CharField(
```

Comment:
There must be, but I don't know how yet. In the next branch (for #142), I've added the `LearningResource.learning_resource_type` to the config, which is a foreign key, and it indexes it fine, despite it being a foreign key. I'm going through the documentation and will figure it out.

The docs seem good content-wise, but the progression is pretty bad. It brings you from zero to 25, then jumps right to 60.